_data/cabellwg/cabellwg.md

```diff
@@ -0,0 +1,7 @@
+---
+name: William Cabell
+institution: Virginia Tech
+profile_pic: not_found.png
+quote: "If one uses the tools of modern algebraic topology, singular homology theory in particular, the proof is quite straightfoward." - James Munkres
```

Comment:
Remove double quotes from begin and end.

Reply:
Quote must **not** be longer than 100 characters, yours is **144** characters.

Reply:
Fixed everything, thank you!